assets/images/invoice-generic.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M6.75 7C6.33579 7 6 7.33579 6 7.75C6 8.16421 6.33579 8.5 6.75 8.5H13.25C13.6642 8.5 14 8.16421 14 7.75C14 7.33579 13.6642 7 13.25 7H6.75Z" fill="#03D47C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M3.8254 1.70739L4.75 2.49991L6.1746 1.27882C6.36185 1.11832 6.63815 1.11832 6.8254 1.27882L8.25 2.49991L9.6746 1.27882C9.86185 1.11832 10.1382 1.11832 10.3254 1.27882L11.75 2.49991L13.1746 1.27882C13.3618 1.11832 13.6382 1.11832 13.8254 1.27882L15.25 2.49991L16.1746 1.70739C16.4989 1.42939 17 1.65984 17 2.08702V7.1875L17.6306 7.69986C17.8643 7.88975 18 8.17484 18 8.47597V18C18 19.1046 17.1046 20 16 20H4C2.89543 20 2 19.1046 2 18V8.47597C2 8.17484 2.1357 7.88975 2.36941 7.69986L3 7.1875V2.08702C3 1.65984 3.50106 1.42939 3.8254 1.70739ZM5 4.91975V9.52781L10 12L15 9.52781V4.91975L13.5 3.63403L11.75 5.13403L10 3.63403L8.25 5.13403L6.5 3.63403L5 4.91975Z" fill="#03D47C"/>
+</svg>
```

Comment:
We usually supply them at pure black (`#000000`) - maybe we just need to edit the fill value here as a quick fix? Otherwise I am happy to provide a new asset.

Reply:
Here is an update, exported the same exact way I've exported many other icons in the repo. 
[invoice-generic.svg.zip](https://github.com/Expensify/App/files/15124398/invoice-generic.svg.zip)

Reply:
Updated to use provided icon - https://github.com/Expensify/App/pull/40015/commits/e22c3eb565a51ad21ddc94eaaba237cce3358a2d